src/db.js

```
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '2907',
```

It's generally not recommended to hardcode credentials, even as a fallback. If the `POSTGRES_PASSWORD` is not set in the environment, it's often better for the application to fail immediately with an explicit error rather than attempting to connect with a default password. This makes configuration issues easier to diagnose.